.travis.yml

```diff
@@ -5,9 +5,11 @@ python:
 before_install:
   - sudo apt-get update -qq
   - sudo apt-get install -qq libdb-dev
+  - mkdir wheelhouse
+  - wget 'http://www.well.ox.ac.uk/~jk/wheelhouse/pysam-0.8.2.1-cp27-none-linux_x86_64.whl' -O 'wheelhouse/pysam-0.8.2.1-cp27-none-linux_x86_64.whl'
```

Comment:
Yeah, using a tarball would enable us to replace the mkdir and wget with a wget and tar -xzvf, which seems cleaner to me (plus enables us to add other wheels in the future without modifying `.travis.yml`, if we ever think that desirable).